db/blip_handler.go

```diff
@@ -216,12 +216,13 @@ func (bh *blipHandler) handleSubChanges(rq *blip.Message) error {
 	bh.continuous = continuous
 	// Start asynchronous changes goroutine
 	go func() {
-		// Pull replication stats by type - Active stats decremented in Close()
+		// Pull replication stats by type - Active continuous stats decremented in Close(), one shot after changes sent
 		if bh.continuous {
 			bh.replicationStats.SubChangesContinuousActive.Add(1)
 			bh.replicationStats.SubChangesContinuousTotal.Add(1)
 		} else {
 			bh.replicationStats.SubChangesOneShotActive.Add(1)
+			defer bh.replicationStats.SubChangesOneShotActive.Add(-1)
```

Comment:
Can you check whether moving the SubChangesContinuousActive decrement here would also work?  Would be nice to keep it all in one place if possible.